tests/rust_usage_test/tests/integration_test.rs

```diff
@@ -15,9 +15,18 @@
  * limitations under the License.
  */
 
+#[macro_use]
 extern crate quickcheck;
-
 extern crate flatbuffers;
+extern crate flexbuffers;
+extern crate rand;
+extern crate serde;
+#[macro_use]
+extern crate serde_derive;
+#[macro_use]
+extern crate quickcheck_derive;
+
+mod flexbuffers_tests;
 
```

Comment:
Please remove the `cargo fmt` changes, it makes it much harder to review this file.

Reply:
Done.